docs/content/querying/dimensionspecs.md

```diff
@@ -195,7 +195,10 @@ Example for the `__time` dimension:
 ```
 
 ### Lookup extraction function
-Explicit lookups allow you to specify a set of keys and values to use when performing the extraction
+
+Lookups are a concept in Druid where dimension values are (optionally) replaced with a new value. 
```

Comment:
should be "replaced with new values".  Because a bunch of values aren't being replaced with a single new value, are they?